holidayextras/holidayextras.github.com
# Remove Careers from Tech Blog

#### What does this PR do? (please provide any background)
- Remove the careers page and instead links to our join.holidayextras.com site.

Reasons for this are:
- Join site is always up to date
- No dev needed to update (cost saving)

#### What tests does this PR have?
None

#### How can this be tested?
Download locally, run `npm i` and `npm start` (this doesn't work for me).
If the above fails run `jekyll serve`
Check changes locally

#### Any tech debt?
Ive remove all know traces of the careers page though there may be stuff I've missed.

#### What gif best describes how you feel about this work?
![food porn](https://media.giphy.com/media/aagX4Bl8Fo21G/giphy.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
